src/server/system_services/master_key_manager.js

```diff
@@ -73,6 +73,17 @@ class MasterKeysManager {
         this.is_initialized = true;
     }
 
+    async update_root_key(new_secret) {
+        const r_key = {
```

Comment:
should we somehow reuse load_root_key()? looks like they have similar logic besides 1/2 lines

Reply:
done